pkg/kubelet/cm/cpumanager/state/state_mem.go

```diff
@@ -23,20 +23,39 @@ import (
 	"k8s.io/kubernetes/pkg/kubelet/cm/cpuset"
 )
 
+type ContainerCpuAssignment map[string]cpuset.CPUSet
```

Comment:
s/ContainerCpuAssignment/ContainerCPUAssignments

and move the type definition from state_mem.go to state.go